src/cli/platform/sgx/crl.rs

```diff
@@ -22,10 +31,43 @@ pub struct CrlCache {}
 
 impl CrlCache {
     pub fn execute(self) -> anyhow::Result<ExitCode> {
-        let dir = sgx_cache_dir()?;
-        save_file(CERT_CRL, &dir)?;
-        save_file(PROCESSOR_CRL, &dir)?;
-        save_file(PLATFORM_CRL, &dir)?;
+        let mut dest_file = sgx_cache_dir()?;
+        dest_file.push("crls.der");
+        let dest_file = dest_file.as_path();
```

Comment:
You don't need this, just use `.open(&dest_file)` in `OpenOptions`

Reply:
This is needed in order to have the path in calls to `.context()`:

```
error[E0382]: borrow of moved value: `dest_file`
  --> src/cli/platform/snp/crl.rs:55:17
   |
31 |         let mut dest_file = sev_cache_dir()?;
   |             ------------- move occurs because `dest_file` has type `PathBuf`, which does not implement the `Copy` trait
...
52 |             .open(dest_file)
   |                   --------- value moved here
...
55 |                 dest_file.as_path()
   |                 ^^^^^^^^^^^^^^^^^^^ value borrowed here after move
```

Reply:
`&`

Reply:
Fixed